draft-js-dnd-plugin/src/modifiers/onDropBlock.js

```diff
@@ -0,0 +1,40 @@
+import AddBlock from './addBlock';
+import RemoveBlock from './removeBlock';
+import { Entity } from 'draft-js';
+
+export default function onDropBlock() {
+  return function onDropBlockInner(e) {
+    const { selection, dataTransfer, editorState, onChange } = e;
+
+    const state = editorState();
```

Comment:
since `editorState` is actually not the editorState, but rather getEditorState I recommend to rename it.

same for `onChange`, better to use `updateEditorState`